src/Native/Unix/System.Security.Cryptography.Native/pal_ssl.h

```diff
@@ -396,3 +392,8 @@ DLLEXPORT int32_t CryptoNative_SslSetTlsExtHostName(SSL* ssl, uint8_t* name);
 Shims the SSL_get_current_cipher and SSL_CIPHER_get_id.
 */
 DLLEXPORT int32_t CryptoNative_SslGetCurrentCipherId(SSL* ssl, int32_t* cipherId);
+
+/*
+Shims the SSL_CIPHER_get_name and SSL_CIPHER_get_version
```

Comment:
```suggestion
Looks up a cipher by the IANA identifier, returns a shared string for the OpenSSL name for the cipher,
and emits a value indicating if the cipher belongs to the SSL2-TLS1.2 list, or the TLS1.3+ list.
```

(Assuming, of course, it's the IANA ID).
It isn't "Shim" because it doesn't just pass the inputs and return the result.